openedx/core/djangoapps/credit/email_utils.py

```diff
@@ -12,7 +12,6 @@
 
 from django.conf import settings
 from django.contrib.auth.models import User
-from django.contrib.staticfiles import finders
```

Comment:
I think you can just change this to:
 ```
from staticfiles import finders
```

That way we don't need to make assumptions in the code about how the static file is located.

(It's very confusing, but edx-platform has two versions of `staticfiles` installed.  One is called `staticfiles` and the other is `django.contrib.staticfiles`.  We want to use `staticfiles`).